execution_workflows/APAtrap/README.md

```diff
@@ -0,0 +1,80 @@
+# APAtrap
+The APAtrap R package does the following three steps: 
+1. Refines annotated 3'UTR and identifies novel 3' UTRs and 3' UTR extensions
+2. Aims to identify all potential APA (alternative polyadenylation) sites
+3. Detects genes with differential APA site usage between conditions by leveraging 
+   the resolution of RNA-seq data
+
+The [paper](https://academic.oup.com/bioinformatics/article/34/11/1841/4816794) is titled APAtrap: identification and quantification of 
+alternative polyadenylation sites from RNA-seq data <br>
+The [application download](https://sourceforge.net/projects/apatrap/files/) is free to download
+and [user manual](https://sourceforge.net/p/apatrap/wiki/User%20Manual/) was used as a referrence
+to create the nextflow pipeline flow of this module
+
+### Steps to run this:
+ - We can either run the workflow to obtain identification and quantification outputs, or differential output. There
+   is currently no way to run the workflow once to get outputs for all three challenges
+ - samplesheet_example_files.csv contains paths to all the input files provided for each workflow run
+ - Check the path to APAtrap with `pwd` and replace the `path_to` in samplesheet_example_files.csv with the 
+   path from the `pwd` command
+ - Each row in the samplesheet contains files for one sample to be processed
+ - Any number of rows (i.e. any number of replicates per condition) can be provided in the samplesheet in any order
+ - If we are running the differential step of APAtrap, make sure to have at least two distinct conditions in the samplesheet and all
+   the rows are processed at the same time
+ - If we are not running the differential step of APAtrap, each condition in the samplesheet will be processed individually 
+   for identification and quantification challenges
+ - Go to `conf/modules.config` to configure the parameters required to run APAtrap workflow. Descriptions of the parameters
+   are located in the file. Head to `Params` for more info
+ - Delete the existing `results` folder to remove results from the previous run, if any. This is to avoid polluting the next
+   run wtih the results from the previous run. This would likely result in an error as well.
+ - Then, you are good to run the pilot benchmark nextflow pipeline with `APAtrap`
+
+```
+nextflow main.nf --input samplesheet_example_files.csv
+```
+
```

Comment:
Could you add a brief explanation to run the differential usage pipeline too please? i.e. Change config value, make sure to delete results/ if exists and the command (or just say same as above).